src/adaptors/http1/http1_client.c

```diff
@@ -678,12 +691,33 @@ static int _client_rx_request_cb(h1_codec_request_state_t *hrs,
     h1_codec_request_state_set_context(hrs, (void *) hreq);
     DEQ_INSERT_TAIL(hconn->requests, &hreq->base);
 
+    //
+    // Start a vanflow record for the client request coming in on the qdr_http1_connection_t.
+    // The parent of this vanflow is the connection's vanflow record.
+    //
+    hreq->base.vflow = vflow_start_record(VFLOW_RECORD_FLOW, hconn->vflow);
+    vflow_set_uint64(hreq->base.vflow, VFLOW_ATTRIBUTE_OCTETS, 0);
+    vflow_add_rate(hreq->base.vflow, VFLOW_ATTRIBUTE_OCTETS, VFLOW_ATTRIBUTE_OCTET_RATE);
```

Comment:
Since there's a rate added to the owning connection flow, I don't think we need a rate on the request/response flows.

Reply:
Done!